salads/fruit-salad.md

```diff
@@ -9,10 +9,11 @@
 - 1 banana, sliced
```

Comment:
Could you add a subtitle for ingredients?

Reply:
Ok, I will do it.
thanks for the suggestion.